tests/test_make.py

```diff
@@ -1587,3 +1587,13 @@ class B(A):
             "Comparision of subclasses using %s is deprecated and will be "
             "removed in 2019." % (op,)
         ) == w.message.args[0]
+
+
+class TestDocs:
+    def test_init_docs(self):
```

Comment:
Please add a docstring that describes what you're testing. [This article](https://jml.io/pages/test-docstrings.html) will show you our best practices in a nice example.